### ISSUE TITLE
Add layers dynamically

### ISSUE THREAD

**Issue by kpoman** [thedirtyfew/dash-leaflet#157]
Hello,
Is there a way to add geojson to an existing map after that geojson data is uploaded to the dash page (add dynamically childs to a Map children property) ? I am looking for a way to upload my geojson defs and have them displayed on the dash-leaflet existing map dynamically (I already receive the geojson data via a callback).
Thank you

**Comment by emilhe** [comment]
Yes, that should be possible. Did you test the code in a non-dynamic context (just to make sure that the generated geojson works)?

**Comment by juanfcocontreras** [comment]
I think the problem here is the following line:

```
@app.callback(Output("layer", "children"),
```

I think it should be like this:

```
@app.callback(Output("map", "children"),
```

If you turn on debug option in Dash (`app.run_server(debug=True)`), you can see these problems.

I think this issue can be closed.
